### ISSUE TITLE
[Screen reader - CosmosDBwebsites - Query Playground] Form elements must have labels (#checkbox-1)

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1503]
User Experience:
The purpose of form elements such as checkboxes, radio buttons, input fields, etcetera, is often apparent to sighted users, even if the form element is not programmatically labeled. Screen readers users require useful form labels to identify form fields

Repro Steps:
1- Navigate to url:- https://www.documentdb.com/sql/demo 
2- Home page will open.​
3- Navigate through all the controls appearing in "Home Page"
4- Run Accessibility Insight for Web and verify the issue. 

Actual result:
Form elements doesn't have labels (#checkbox-1)

Expected result:
Form elements must have labels (#checkbox-1)

Note:
Same issue exists for all tab elements.

MAS References:
MAS 1.3.1 - Info and Relationships (40)

Issue: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI)

Target application title: Azure Cosmos DB Query Playground​
Target application url: https://www.documentdb.com/sql/demo​
​
Element path: h3:nth-child(2)​
​
Snippet: h3 Learn more about Azure Cosmos DB’s rich querying over schema-free JSON data. Azure Cosmos DB is Microsoft’s globally-distributed, multi-model database service. h3
​
How to fix:​
Fix any of the following:​
  Element has insufficient color contrast of 3 (foreground color: #ffffff, background color: #48a0b8, font size: 13.0pt, font weight: normal). Expected contrast ratio of 4.5:1​
​
This accessibility issue was found using Accessibility Insights for Web 2.0.4 (axe-core 3.1.1), a tool that helps find and fix accessibility issues
![MAS1 3 1_Form elements must have labels (#checkbox-1)](https://user-images.githubusercontent.com/30456910/60590663-2d3d9800-9dba-11e9-92e1-2d0aa879d6f9.jpg)
. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by chrisdias** [comment]
this does not appear to be a Monaco editor issue, this looks like a problem with the documentdb site.

**Comment by isidorn** [comment]
@sonimishra hi if you think this is an issue in the monaco editor please file the issue in concise steps in humand readable form only mentioning the monaco editor. Do not mention the Azure Stream Analyitics in your reproducable steps. Please put some effor in this.
Thank you for your understanding.

**Comment by sonimishra** [comment]
This is indeed external.
![image](https://user-images.githubusercontent.com/30456910/66739548-98553b00-ee8e-11e9-8be6-a3b8cfd2e6b5.png)

**Comment by sonimishra** [comment]
This is indeed external.
![image](https://user-images.githubusercontent.com/30456910/67942503-f9ef0680-fbfd-11e9-9ac4-09f897fab151.png)
